prowler/__main__.py

```diff
@@ -332,18 +332,21 @@ def prowler():
     # Outputs
     # TODO: this part is needed since the checks generates a Check_Report_XXX and the output uses Finding
     # This will be refactored for the outputs generate directly the Finding
-    finding_outputs = [
-        Finding.generate_output(global_provider, finding, output_options)
-        for finding in findings
-    ]
+    finding_outputs = []
+    for finding in findings:
+        try:
+            finding_outputs.append(
+                Finding.generate_output(global_provider, finding, output_options)
+            )
+        except Exception:
+            continue
```

Comment:
Maybe we can add some information, and remove the `continue`. What do you think?

```
logger.warning(f"Failed to generate output for finding '{finding}': {exc}")
```

Reply:
I added that log line within the `generate_output` `try/except` clause, if it is not enough I can add another one here.

Reply:
I didn't notice, its ok for me